lib/chef/provisioning/vsphere_driver/driver.rb

```diff
@@ -325,6 +325,7 @@ def attempt_ip(machine_options, action_handler, vm, machine_spec)
 
     def wait_for_domain(bootstrap_options, vm, machine_spec, action_handler)
       return unless bootstrap_options[:customization_spec]
+      return unless bootstrap_options[:customization_spec].is_a?(Hash) || bootstrap_options[:customization_spec].is_a?(Cheffish::MergedConfig)
```

Comment:
if customizing a windows box, I'm not sure this would be a good idea. Perhaps there is a way to get the vmware customization object and inspect for a domain?